### PR TITLE
56 game page answer gpt box

### PR DESCRIPTION
**Pull request by bjohnson66**
# Implement Live Updates, ChatGPT Integration, and UI Enhancements

## Summary of Changes

1. **Real-time Updates with Ruby Channels:**
   - Implemented Ruby Channels to enable live updates for users in the lobby.
   - Real-time updates for users joining or leaving the lobby.
   
2. **ChatGPT Integration:**
   - Added initial ChatGPT functionality for interactive chat within the app.
   - Integrated basic natural language processing for user queries.
   
3. **Friend Management Enhancements:**
   - Introduced a **Remove Friend** button for users to manage their friends list.
   - Added **Profile Pictures** to the Friend page for better visual representation.
   
4. **Settings and Game Page Visual Overhaul:**
   - Changed the background image of the **Settings** page to enhance aesthetics.
   - Added a **Navigation Bar** to the **Game Page** to improve user experience and navigation.
   - Major visual overhaul of the **Game Page** to improve layout and design.
   
### Images:

- **Live Updates in Lobby (Users Joining), GPT Exmample shows up on all games:**
  
  
image

-**Fiends page**

image

---

### CODE REVIEW

File: features/player_name.feature [hunk 1]
```diff
@@ -18,4 +18,5 @@ Feature: Displaying player profile name in the game
     And I fill in "join_game_join_code" with "A1B2C3"
     And I submit the join game form
     Then I should see "You have successfully joined the game."
-    And my profile name "ExampleUser" should be displayed in the game lobby
+    # And my profile name "ExampleUser" should be displayed in the game lobby
```
Comment (halimyu) [inline features/player_name.feature:21]: hmm, I think u have some pending stuff

File: features/player_name.feature [hunk 2]
```diff
@@ -18,4 +18,5 @@ Feature: Displaying player profile name in the game
...
+    Given PENDING SCENARIO
```
Comment (halimyu) [inline features/player_name.feature:22]: hmm, I think u have some pending stuff

File: features/step_definitions/web_steps.rb [hunk 3]
```diff
@@ -204,3 +204,7 @@ def with_scope(locator)
   click_button "Sign in"
   @current_user = User.find_by(email: email)
 end
+
+Given("PENDING SCENARIO") do
```
Comment (halimyu) [inline features/step_definitions/web_steps.rb:208]: here too

**Review by halimyu — CHANGES_REQUESTED** [review]
change to devel branch

**Comment by bjohnson66** [comment]
DONT

**Comment by bjohnson66** [comment]
ITS BROKE

**Pull request closed without merging**